cves/CVE-2015-5964.yml

```diff
@@ -290,7 +292,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: Lack of thorough red teaming on part of the developers. This sort of thing might have been discovered if they intentionaly tried to break their software more.
```

Comment:
While testing of the software might have revealed this issue, I believe testing from an outside party may have been more helpful as this is seems to be an issue likely overlooked by the initial development team.

Reply:
That would work too, sure.